### ISSUE TITLE
Footnote 12

### ISSUE THREAD

**Issue by basma-b** [yangliuy/NeuralResponseRanking#1]
Please, what do you mean by footnote 12 ?
> The reported SMN results with the code from authors are on the raw data sets of UDC and MSDialog without any over sampling of negative training data

**Comment by yangliuy** [comment]
You can take a look at the code at line 146 to line 156 of 

https://github.com/MarkWuNLP/MultiTurnResponseSelection/blob/master/tensorflow_src/SCN.PY

You can understand that the authors of SMN did negative data sampling during model training stage. The reported SMN results in our paper are the results without any over sampling of negative training data as did in the ACL'17 paper. We also didn't do over sampling of negative training data for DMN-PRF and DMN-KD. So this is a fair comparison. We used the raw training data of UDC and MSDialog for model training.

**Comment by basma-b** [comment]
So if I understand, you train your model only on positive samples (context, utterance, label = 1) ? or you take the training samples as they are given in the corpus positives and negatives ?

**Comment by yangliuy** [comment]
I train my model with both the given positive samples and given negative samples in the data. The data statistics can be found in Table 4.
